src/sidebar/markdown-commands.js

```diff
@@ -83,7 +83,7 @@ function replaceText(state, pos, length, text) {
  * Convert the selected text into a Markdown link.
  *
  * @param {EditorState} state - The current state of the input field.
- * @param {LinkType} linkType - The type of link to insert.
+ * @param {LinkType} [linkType] - The type of link to insert.
```

Comment:
Side note - There is an ES5-style default value initialization inside `convertSelectionToLink` that we could modernize.

Reply:
done